index.ios.js

```diff
@@ -4,7 +4,8 @@ import { AppRegistry, Text, View } from 'react-native';
 import {
   landingPage,
   login,
+  poll,
```

Comment:
Since we're making an effort to change component names to capital, let's start now